### PR TITLE
MorphingCircle Outline

### PR DESCRIPTION
**Pull request by mlohr**
Added support for optional outline for MorphingCircle

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Code for https://alexdremov.me/swiftui-advanced-animation/
 
-
+Added support for optional outline
```
Comment (alexdremov) [inline README.md:5]: - [ ] Don't think that this line needs to be in README

File: README.md [hunk 2]
```diff
@@ -14,7 +14,8 @@ import MorphingShapes
 struct ContentView: View {
     var body: some View {
         VStack {
-            MorphingCircle()
+            // added support for optional outline:
+            MorphingCircle(outlineColor: .orange, outlineWidth: 10.0)
```
Comment (alexdremov) [inline README.md:18]: - [ ] Same thing. I'd suggest keeping README simple

File: Sources/MorphingShapes/MorphingBackground.swift [hunk 3]
```diff
@@ -9,6 +9,7 @@ import SwiftUI
 import Foundation
 import Combine
 
+@available(macOS 11.0, *)
```
Comment (alexdremov) [inline Sources/MorphingShapes/MorphingBackground.swift:12]: - [ ] Currently, it's version 10.15 in `Package.swift`. Are you sure you need to set macOS version this high?

File: Sources/MorphingShapes/MorphingCircle.swift [hunk 4]
```diff
@@ -94,14 +95,19 @@ public struct MorphingCircle: View & Identifiable & Hashable {
     let outerSize: CGFloat
     var color: Color
     let morphingRange: CGFloat
+    ///
+    var outlineColor: Color
```
Comment (alexdremov) [inline Sources/MorphingShapes/MorphingCircle.swift:99]: - [ ] Outline is never changed, use `let`

File: Sources/MorphingShapes/MorphingCircle.swift [hunk 5]
```diff
@@ -117,9 +123,11 @@ public struct MorphingCircle: View & Identifiable & Hashable {
         
     }
     
-    public init(_ size:CGFloat = 300, morphingRange: CGFloat = 30, color: Color = .red, points: Int = 4,  duration: Double = 5.0, secting: Double = 2) {
+    public init(_ size:CGFloat = 300, morphingRange: CGFloat = 30, color: Color = .red, outlineColor: Color = .clear, outlineWidth: Double = 2, points: Int = 4,  duration: Double = 5.0, secting: Double = 2) {
```
Comment (alexdremov) [inline Sources/MorphingShapes/MorphingCircle.swift:126]: - [ ] For compatibility reasons, parameters should be added at the end

**Review by alexdremov — CHANGES_REQUESTED** [review]

**Comment by mlohr** [comment]
Will submit new pull request with requested changes

**Comment by mlohr** [comment]
Thanks I have made updates and put in a new pull request

Op do 13 okt. 2022 om 21:56 schreef Aleksandr Dremov <
***@***.***>:

**Pull request closed without merging**